pyspecProcScripts/integral_w_error.py

```diff
@@ -53,9 +56,12 @@ def integral_w_errors(
         kwargs = {"convolve_method": convolve_method}
     else:
         kwargs = {}
-    frq_slice = integrate_limits(
-        select_pathway(s, sig_path), convolve_method=convolve_method, cutoff=cutoff, fl=fl
-    )
+    if frq_slice is not None:
+        frq_slice = frq_slice 
+    else:    
+        frq_slice = integrate_limits(
+            select_pathway(s, sig_path), convolve_method=convolve_method, cutoff=cutoff, fl=fl
+        )
```

Comment:
Again default is to use automation but with option of frq_slice the user can feed it a slice they wish to integrate over - useful for integrating/testing noise or comparing two datasets exactly